docs/api/authentication.rst

```diff
@@ -0,0 +1,61 @@
+##############
+Authentication
+##############
+
+.. _authentication:
+
+We propose a setup using :term:`Firefox Accounts`, that verifies
```

Comment:
Yes, we should probably fix it here too then: http://cliquet.readthedocs.org/en/latest/api/authentication.html#authentication